pkg/miniogw/gateway-storj.go

```diff
@@ -72,48 +204,104 @@ func (s *storjObjects) GetObject(ctx context.Context, bucket, object string,
 }
 
 func (s *storjObjects) GetObjectInfo(ctx context.Context, bucket,
-	object string) (objInfo cmd.ObjectInfo, err error) {
+	object string) (objInfo minio.ObjectInfo, err error) {
 	panic("TODO")
 }
 
 func (s *storjObjects) ListBuckets(ctx context.Context) (
-	buckets []cmd.BucketInfo, err error) {
-	return []cmd.BucketInfo{{
-		Name:    "test-bucket",
-		Created: time.Now(),
-	}}, nil
+	buckets []minio.BucketInfo, err error) {
+	return s.getBuckets()
 }
 
 func (s *storjObjects) ListObjects(ctx context.Context, bucket, prefix, marker,
-	delimiter string, maxKeys int) (result cmd.ListObjectsInfo, err error) {
-	return cmd.ListObjectsInfo{
-		IsTruncated: false,
-		Objects: []cmd.ObjectInfo{{
-			Bucket:      "test-bucket",
-			Name:        "test-file",
-			ModTime:     time.Now(),
-			Size:        0,
-			IsDir:       false,
-			ContentType: "application/octet-stream",
-		}},
-	}, nil
+	delimiter string, maxKeys int) (result minio.ListObjectsInfo, err error) {
+	return s.getFiles(bucket)
 }
 
 func (s *storjObjects) MakeBucketWithLocation(ctx context.Context,
 	bucket string, location string) error {
 	panic("TODO")
 }
 
+//encryptFile encrypts the uploaded files
+func encryptFile(data io.ReadCloser, blockSize uint, bucket, object string) error {
+	dir := os.TempDir()
+	dir = dir + "gateway/" + bucket + "/" + object
+	fmt.Println("temp dir = ", dir)
```

Comment:
Don't forget to remove this before committing, or use the log package if that log message is important in future.

Reply:
That I missed to take it out. I will remove it